packages/ndla-ui/src/locale/messages-sma.ts

```diff
@@ -552,6 +552,7 @@ const messages = {
       files: 'Fijlh',
       embedlink: 'Sïjsebïejemesvaalhtese',
       concept: 'Buerkiestimmieh',
+      gloss: 'Gloser',
```

Comment:
Antar denne skal være slik
```suggestion
      gloss: 'Glosa',
```